bigquery/api/BigquerySample/Snippets.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Diagnostics;
+using Google.Cloud.BigQuery.V2;
+
+namespace GoogleCloudSamples
+{
+    public static class BiqQuerySnippets
+    {
+		/// <summary>
+        /// Loads the table from a JSON source.
+        /// </summary>
+        /// <param name="client">BigQuery client</param>
+		/// <param name="datasetId"></param>
+		/// <param name="tableId"></param>
+		/// <param name="gcsURI"></param>
+		/// <param name="field1"></param>
+		/// <param name="field2"></param>
+        public static void LoadTableFromJSON(BigQueryClient client,
+		    string datasetId, string tableId, string gcsURI, string field1,
```

Comment:
Could you use the concrete URI in the sample itself instead of just via the test?